### PR TITLE
don't allow to delete other persons chats in 1-1 chats

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob Röhrl <jakob.roehrl@web.de>
solves #5290

But unfortually it doesn't work. 
Who can help?

### CODE REVIEW

**Comment by PVince81** [comment]
- [ ] also disallow on backend

**Comment by nickvergessen** [comment]
https://github.com/nextcloud/spreed/blob/c39f2c7ce93ffc0b6de2c72fc84094eff738d0f8/lib/Controller/ChatController.php#L554-L559

**Comment by PVince81** [comment]
@nickvergessen seems the backend bits are not working correctly

1. Checkout this PR
2. Setup 1-1 chat
3. Revert the changes in the UI to make the delete action accessible
4. Refresh page
5. Delete message from other participant

Expected: 403
Actual: still deleted

File: lib/Controller/ChatController.php [hunk 1]
```diff
@@ -551,9 +551,11 @@ public function deleteMessage(int $messageId): DataResponse {
 		}
 
 		$attendee = $this->participant->getAttendee();
-		if (!$this->participant->hasModeratorPermissions(false)
-			&& ($message->getActorType() !== $attendee->getActorType()
-				|| $message->getActorId() !== $attendee->getActorId())) {
+		$isOwnMessage = $message->getActorType() === $attendee->getActorType()
+			|| $message->getActorId() === $attendee->getActorId();
```
Comment (PVince81) [inline lib/Controller/ChatController.php:555]: shouldn't this be `&&` ?

Comment (nickvergessen) [reply lib/Controller/ChatController.php]: yeah the integration test also failed, it works now :D

**Review by PVince81 — APPROVED** [review]
Tested, works :+1:

**Comment by nickvergessen** [comment]
chat integration tests passed

**Pull request merged**